### ISSUE TITLE
Windows Defender found malware infection - Pencil 3.0.0RC1

### ISSUE THREAD

**Issue by ZenithDK** [evolus/pencil#85]
I just downloaded the RC1, and immediately after running the executable I got a warning from Windows Defender about it finding malware:
https://www.microsoft.com/security/portal/threat/encyclopedia/entry.aspx?name=Trojan%3aWin32%2fVarpes.M!cl&threatid=2147708972&enterprise=1

It didn't specifically state that the malware originated from the Pencil binary, but it was in a file in my user's AppData/Temp - which seems to indicate that it would come from the just unpacked Pencil RC1 executable?

Could you please check to make sure the website has not been breached and the download is infected?

**Comment by gigios** [comment]
I have the same issue.

**Comment by ZenithDK** [comment]
I'm on a corporate network so I'm unsure if they have already rolled that
out. It could be legit or just a false positive I guess.

**Comment by awesomektvn** [comment]
I have the same issue. Is it a false positive ?

**Comment by mbrainiac** [comment]
#76

**Comment by gigios** [comment]
I think that it's a false positive.
In my case each time a start the Pencil, in my temporary folder is created on (or more) temporary files and one of this is identified as "Win32/Varpes.M!cl" malware.
For example now I have the temporary file "DF21.tmp.exe" blocked by Defender, but if I stop the defender and send this file to VirusTotal to check if it's a real malware, the result is that all the AntiVirus used in the VirusTotal portal does not detect any problem in the file!
For me this is a false positive, the only big problem is that I'm not able to add the Pencil to the allowed programs in the Defender, this because the filename change at each start.

**Comment by ZenithDK** [comment]
This is a duplicate of #85, closing.
